Package.swift

```diff
@@ -17,6 +17,9 @@ import PackageDescription
 
 let package = Package(
     name: "swift-nio-http2",
+    platforms: [
+        .macOS("10.15")
+    ],
```

Comment:
We can't add this; it's an API breaking change. You can use `@available` guards in the code though.